docs/articles/documentation/extending-testcafe/reporter-plugin/README.md

```diff
@@ -81,14 +85,14 @@ export default function () {
 }
 ```
 
-These methods should output the desired information at certain moments during the test run.
-All the required data is provided for these methods through their parameters.
+TestCafe calls these methods during the test run. Use data they receive through their parameters to compose a report.
+See [Reporter Methods](reporter-methods.md) for API description and examples.
 
-To output this information, use [helper methods and libraries](helpers.md).
-TestCafe will mix in the helper methods to the reporter, so that you can access the helper methods by using `this`.
+To output the report, use [helper methods and libraries](helpers.md).
+TestCafe mixes in the helper methods to the reporter, so that you can access them with `this`.
```

Comment:
no idea what this sentence says
mixes in methods to the reporter???

Reply:
**Mixing in** means adding members to objects without inheritance.
The most widely used term is a noun - **mixin**. It means a class/object whose members are added to other objects via 'mixing in'. See https://en.wikipedia.org/wiki/Mixin.

As long as we're not talking about objects (mixins) here, but rather about the mechanism, I used the verb 'mix in'. It's not as frequent as the noun but I think it is well comprehensible.

Anyway, I guess I wrote this sentence in the wrong way, it should be:

*TestCafe mixes the helper methods into the reporter. You can access them with `this`.*

Reply:
yes that last rewrite is  fine